src/products/filters/models/ProductFilter.ts

```diff
@@ -0,0 +1,16 @@
+import AlphabeticCurrencyCode from '../../../model/AlphabeticCurrencyCode';
+import ChannelApeApiError from '../../../model/ChannelApeApiError';
+
+export default interface ProductFilter {
+  alphabeticCurrencyCode: AlphabeticCurrencyCode;
```

Comment:
I know our docs are a little lacking in response bodies. But a full list of fields for the response are:
If it's trivial can you please add these? 
`
        String id
	String businessId
	String alphabeticCurrencyCode
	String label
	String title
	Set<String> vendors
	Set<String> skus
	Set<String> upcs
	String condition
	String primaryCategory
	String secondaryCategory
	Set<String> tags
	String weightMin
	String weightMax
	String retailMin
	String retailMax
	String wholesaleMin
	String wholesaleMax
	Long quantityMin
	boolean complement
	Long quantityMax
	String updatedAt
	String createdAt

Reply:
@ryankazokas While I added this to the response body here, I think you might have meant request body (ProductFilterRequest)?
I was just looking at the request for creating a product filter and saw these same fields in the request body, not the response.

Reply:
I meant response. We are just accepting the business id right now on the request for create? If we can add these to the request body without much work, then you can go for it. Here are the fields that we accept on create:

`
String businessId;
	String label;
	String title;
	Set<String> vendors = new HashSet<>();
	Set<String> skus = new HashSet<>();
	Set<String> upcs = new HashSet<>();
	String condition;
	String primaryCategory;
	String secondaryCategory;
	Set<String> tags = new HashSet<>();
	String weightMin;
	String weightMax;
	String retailMin;
	String retailMax;
	String wholesaleMin;
	String wholesaleMax;
	Long quantityMin;
	Long quantityMax;
	boolean complement;
`

Reply:
![image](https://user-images.githubusercontent.com/41341349/62872217-e0f74900-bcd1-11e9-95dc-17f5d1ca4691.png)

Why doesn't the REST API return those fields (from your first comment) in the response when creating a filter?

I won't worry about adding those extra filter request fields if we don't need them yet.  I just wanted to make sure the response object is correct.

Reply:
We don't return null values. You'll see this on all of our rest services. The sku's/tags/upc's/etc are all empty. So that's why you see those. The fields i sent you are the actual fields that could be returned.

Reply:
Understood, thanks for the clarification.